### DISCUSSION TITLE
Add copy button to code snippets

### DISCUSSION THREAD

**Discussion by GZLiew** [shuding/nextra#851] [Ideas]
Hi Great blog tool this is

I was wondering can we have a copy button for code snippets not sure how difficult is it to add one?

**Comment by jermanuts** [comment]
it's already in v2 docs (beta), https://nextra-v2-git-shu-agay-shuding1.vercel.app/docs/guide/markdown hover over code snippet you will see on the top right a copy button.

For the v2 docs you can build it from the core repo. https://github.com/shuding/nextra/tree/core/docs

**Reply by GZLiew** [reply]
ah I see it I'm still on v1 since last year

**Reply by GZLiew** [reply]
Will the migration be easy from v1 to v2?
